Examples/UIExplorer/TextExample.windows.js

```diff
@@ -285,6 +367,57 @@ exports.examples = [
     );
   },
 }, {
+  title: 'backgroundColor attribute',
```

Comment:
See comment in ReactTextViewManager, I don't think we should support 'backgroundColor' if it's not native on the TextBlock.